app/src/main/res/layout/item_join_domain.xml

```diff
@@ -1,170 +1,170 @@
-                        android:textSize="@dimen/sp_15"
-                        android:textStyle="bold"
-                        />
-                    <TextView
-                        android:layout_width="wrap_content"
+                    <LinearLayout
+                        android:layout_width="0dp"
                         android:layout_height="wrap_content"
-                        android:layout_marginLeft="@dimen/dp_4"
-                        android:layout_marginStart="@dimen/dp_4"
-                        android:textColor="@color/black"
-                        android:textSize="@dimen/sp_15"
-                        android:textStyle="bold"
-                        />
-                </LinearLayout>
+                        android:layout_weight="2"
+                        android:orientation="vertical"
+                        tools:ignore="Suspicious0dp"
+                        >
+                        <LinearLayout
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:orientation="horizontal"
+                            >
+                            <TextView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:layout_marginLeft="@dimen/dp_4"
+                                android:layout_marginStart="@dimen/dp_4"
+                                android:text="@{`User : ` + viewModel.numberUser}"
+                                android:textColor="@color/black"
+                                android:textSize="@dimen/sp_15"
+                                android:textStyle="bold"
+                                />
+                        </LinearLayout>
 
-                <LinearLayout
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/dp_5"
-                    android:orientation="horizontal"
-                    >
-                    <TextView
-                        android:layout_width="wrap_content"
+                        <LinearLayout
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_marginTop="@dimen/dp_5"
+                            android:orientation="horizontal"
+                            >
+                            <TextView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:layout_marginLeft="@dimen/dp_4"
+                                android:layout_marginStart="@dimen/dp_4"
+                                android:text="@{`Product : ` + viewModel.numberProduct}"
```

Comment:
hardcode

you can define in ViewModel and use in here
```java
retun getString(R.string.product,numberProduct)
```
``R.string.product`` 's value  is ``Product : %s``